packages/core/protocols/src/proto/dxos/config.proto

```diff
@@ -310,36 +262,28 @@ message Runtime {
       optional string credential = 3;
     }
 
-    message Machine {
```

Comment:
this is something specific to DO auto-deployment process
i think those should be rather obtained from ENV;
we could rework this part when we figure out our "KUBE droplet template" story